client/client.go

```diff
@@ -250,6 +251,27 @@ func parseCertificate(b64 string) (*x509.Certificate, error) {
 	return x509.ParseCertificate(certDER)
 }
 
+// GetProofByHash performs a get-proof-by-hash request, with parameters hash and
+// treeSize.
+// Returned is:
+//   - a GetProofByHashResponse struct, if no error is returned.
+//   - the HTTPData struct returned by GetAndParse() (see above).
+//   - an error, which could be any of the error types returned by
+//     GetAndParse().
+func (lc *LogClient) GetProofByHash(hash string, treeSize uint64) (*ct.GetProofByHashResponse, *HTTPData, error) {
```

Comment:
Hashes are handled with `string`, rather than `[]byte`? Seems suspicious to me...

Reply:
Changed.